pcsx2-qt/Settings/InputBindingWidget.cpp

```diff
@@ -74,9 +74,15 @@ void InputBindingWidget::initialize(
 
 void InputBindingWidget::updateText()
 {
+	const QString binding_tip(tr("\n\nLeft click to assign a new button\nShift + left click for additional bindings"));
+	const QString binding_clear_tip(tr("\nRight click to clear binding"));
+
 	if (m_bindings.empty())
 	{
 		setText(QString());
+
+		const QString no_binding(QString::fromStdString("No bindings registered"));
+		setToolTip(no_binding + binding_tip);
```

Comment:
Should be translatable.

```suggestion
		setToolTip(tr("No bindings registered.") + binding_tip);
```

Reply:
Done :)